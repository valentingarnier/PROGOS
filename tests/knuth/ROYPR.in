5 K
1 0
2 1
3 0
2 2
5 0
