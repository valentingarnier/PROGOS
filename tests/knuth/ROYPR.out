What size (2, 3, 4, 5)? What strategy ([B]rute force, B[i]tfield, [K]nuth)? Y Y Y B B
Please score attempt (positions, colors): R R Y G G
Please score attempt (positions, colors): R G Y P P
Please score attempt (positions, colors): O R Y P Y
Please score attempt (positions, colors): R O Y P R
Please score attempt (positions, colors): Found solution: R O Y P R
